practicaenclases/Practica.py

```diff
@@ -0,0 +1,103 @@
+print "--------------1----------------------"
+
+
+def factorial(num):
+    if num > 0:
+        i = 1
+        Result = 1
+        while i <= num:
+            Result = Result * i
+            i += 1
+        return Result
+
+
+print factorial(0)
```

Comment:
remove this, you are using unit test to test your method

Reply:
commit e6f6b4c6f43732aa5fb9731fbf56be44dd717897